descholar-ceo/AnnounceIT
# #170773943 User sees profilepage

#### What does this PR ​ do​ ?
User should be able to navigate through all features of the application on his/her profile page

#### Description of Task to be completed?
* User signup successfully and he/she immediately get redirected to his/her profile page
* User clicks on logout button and he/she goes back to the signin page

 #### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

*Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/user-profile.html`, will be redirected to the and click on signup button, you will be immediatelly to the user profile-page

* Or if you don't use vscode as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `user-profile.html` and right click and open with your preferred browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)
#### What are the relevant pivotal tracker stories?
[170773943](https://www.pivotaltracker.com/n/projects/2429498)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-18 15-20-57](https://user-images.githubusercontent.com/56886064/72664431-7a0b5800-3a06-11ea-911d-a41501d8d9cd.png)
![Screenshot from 2020-01-18 15-21-13](https://user-images.githubusercontent.com/56886064/72664432-7a0b5800-3a06-11ea-9f69-974b6130bbcd.png)

#### Questions:
No Question
